scripts/task.ts

```diff
@@ -316,11 +318,16 @@ async function runTask(task: Task, details: TemplateDetails, optionValues: Passe
   }
 }
 
+const controllers: AbortController[] = [];
+
 async function run() {
   const allOptionValues = await getOptionsOrPrompt('yarn task', options);
 
   const { task: taskKey, startFrom, junit, ...optionValues } = allOptionValues;
 
+  // In case other tasks need to know
+  process.env.SELECTED_TASK = taskKey;
```

Comment:
Can we pass this as part of `TemplateDetails` rather than using a global?

Reply:
We could, though the select task has nothing to do with the template, right